app/configurations/config.oulu.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable */
+import mergeWith from 'lodash/mergeWith';
+
+const CONFIG = process.env.CONFIG || 'oulu';
+const APP_DESCRIPTION =
+  'Reittiopas uudistuu. Tule mukaan! Ota uuden uuden sukupolven matkaopas käyttöösi.';
+
+const walttiConfig = require('./waltti').default;
+
+export default mergeWith({}, walttiConfig, {
+  CONFIG,
+
+  appBarLink: { name: 'Oulu', href: 'http://www.oulunkaupunki.fi/' },
+
+  colors: {
+    primary: '#c40065',
+  },
+
+  socialMedia: {
+    title: 'Uusi Reittiopas - Oulu',
+    description: APP_DESCRIPTION,
+  },
+
+  title: 'Oulu Digitransit',
```

Comment:
The name Digitransit, should be not visible to customers, we should check all configs for this.